addon/styles/_frost-link.scss

```diff
@@ -121,6 +121,7 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
   &.tertiary {
     background-color: inherit;
     color: $frost-color-blue-1;
+    margin-bottom: -7px;
```

Comment:
Just curious, but what does this provide?  I believe tertiary is completely inline with text now and what Quincy had demoed to me looked right